packages/components/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ### Bug Fix
 
+-   Fix alignment of `BorderStyleControl` legend with border width control's label ([#37244](https://github.com/WordPress/gutenberg/pull/37244)).
```

Comment:
The `BorderStyleControl` component doesn't belong to the `@wordpress/components` package, and therefore we should not add a CHANGELOG entry there for this PR.

Reply:
Thanks! Removed.